### PR TITLE
SSCS-9807: Child Support: e2e Tests - Subscription

### PR DESCRIPTION
**Pull request by rasikasri**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-9807

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: e2e/features/step_definitions/other-party-subscription.steps.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    await anyCcdPage.setValueByElementId('subscriptions_jointPartySubscription_tya', 'jointParty123')
+    await anyCcdPage.setValueByElementId('subscriptions_jointPartySubscription_email', 'jointparty-test@mailinator.com')
+    await anyCcdPage.setValueByElementId('subscriptions_jointPartySubscription_mobile', '01234567890')
+
+    await anyCcdPage.clickElementById('subscriptions_supporterSubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.setValueByElementId('subscriptions_supporterSubscription_tya', 'supportParty123')
+    await anyCcdPage.setValueByElementId('subscriptions_supporterSubscription_email', 'supportparty-test@mailinator.com')
+    await anyCcdPage.setValueByElementId('subscriptions_supporterSubscription_mobile', '01234567890')
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartySubscription_wantSmsNotifications_' + action);
```
Comment (ryandac) [inline e2e/features/step_definitions/other-party-subscription.steps.ts:40]: the formatting is not correct either on this line or the lines above.

File: e2e/features/step_definitions/other-party-subscription.steps.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+     await anyCcdPage.clickElementById('otherParties_0_otherPartySubscription_subscribeEmail_' + action);
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyAppointeeSubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyAppointeeSubscription_subscribeEmail_' + action);
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyRepresentativeSubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyRepresentativeSubscription_subscribeEmail_' + action);
+
+ }
```
Comment (ryandac) [inline e2e/features/step_definitions/other-party-subscription.steps.ts:81]: Incorrect formatting, leads me to believe this is the end of the function.

File: e2e/features/step_definitions/other-party-subscription.steps.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyAppointeeSubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyAppointeeSubscription_tya', 'otherPartyAppointee123')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyAppointeeSubscription_email', 'otherpartyAppontee-test@mailinator.com')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyAppointeeSubscription_mobile', '01234567890')
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyRepresentativeSubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyRepresentativeSubscription_tya', 'otherParty123')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyRepresentativeSubscription_email', 'otherparty-test@mailinator.com')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyRepresentativeSubscription_mobile', '01234567890')
+ } else {
```
Comment (ryandac) [inline e2e/features/step_definitions/other-party-subscription.steps.ts:56]: The No action is not being used. Is it nice to have this code?

**Review by ryandac — CHANGES_REQUESTED** [review]
Correct the formatting and run `yarn lint`

File: e2e/features/step_definitions/other-party-subscription.steps.ts (hunk 1 above)
Comment (rasikasri) [reply e2e/features/step_definitions/other-party-subscription.steps.ts]: Done

File: e2e/features/step_definitions/other-party-subscription.steps.ts (hunk 2 above)
Comment (rasikasri) [reply e2e/features/step_definitions/other-party-subscription.steps.ts:81]: Done

File: e2e/features/step_definitions/other-party-subscription.steps.ts (hunk 3 above)
Comment (rasikasri) [reply e2e/features/step_definitions/other-party-subscription.steps.ts]: Added missing test case to set subscription to No

**Review by ryandac — APPROVED** [review]

**Pull request merged**